registry/network/functions.go

```diff
@@ -0,0 +1,521 @@
+package network
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"net"
+)
+
+// ParseIP parses a string representation of an IP address and returns its net.IP form.
```

Comment:
```suggestion
// ParseIP parses a string representation of an IP address and returns its [net.IP] form.
```

I think this I'd the way to link the godoc

Could you try locally and update the comment of each method if it works?